python/setup.py

```diff
@@ -13,7 +13,9 @@
     name='lib_i2s',
     packages=setuptools.find_packages(),
     install_requires=[
-        'flake8~=3.8',
```

Comment:
why are these dependencies now needed?

Reply:
So as I understand (and this is the same as in https://github.com/xmos/lib_logging/pull/70) the dependencies here should match the requirements.txt because that way if installing this repository as an editable module setuptools knows what the requirements are.

I don't actually know if this is the right way to be doing this, and Python packaging style has moved on significantly since this (should be using pyproject.toml in an ideal world in my opinion) but I'm keeping current style. I think this is an oversight on the part of whomever added pytest and pytest-xdist to the requirements.txt originally, as they also did not pin the version numbers (which again I'm personally unconvinced by doing but it's in keeping with style with the rest of our repos so I've kept it.